.gitignore

```diff
@@ -144,3 +144,6 @@ node_modules/
 /playground/**/.gitignore
 /playground/**/azure.yaml
 /playground/**/next-steps.md
+
+# Ignore mono files on linux
+.mono/
```

Comment:
where do these come from?

Reply:
if you make run `dotnet build` on linux they apear

Reply:
![image](https://github.com/dotnet/aspire/assets/11291885/5769aa56-ed78-485b-a77c-3009dab13da4)

it comes from the devkit as it looks like

Reply:
OK fine.